### PR TITLE
[RTM] Club Rework: Holobar and Club Artist

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Implements Holobar, turning the entire main floor of the bar into a Holodeck.
Also buffs holofloor, implements holoholes, and other QOL, such as new holo stools (that you can break on others' backs).
Contains a set of relevant fixes and tweaks to resolve the errors having the holodeck in such a central place brings, mainly related to power.
The holodeck draws power through its console, from the APC located in the room the console is.

Adds Club Artist in place of the conference room, removing the old Guild Artist and turning their office into maint. Like the Club Workers, they receive no pay by default and must pester another Club Worker or the Manager (both of those roles have access to the account).

## Why It's Good For The Game

Holvr's design documentation.

## Testing

Ran on local, tested from multiple angles with the involvement of a wrench. No bugs left unresolved.

## Changelog
:cl:
add: Holobar
add: Club Artist
remove: Guild Artist
fix: Some wire runtimes fixed when the floor changes types
/:cl:

### CODE REVIEW

**Comment by Holvr0** [comment]
My name is Holvr and I approve of this PR.

**Comment by Mycah142** [comment]
My name isn't Holvr and I voted for Nader.

**Comment by SaltyOldBull** [comment]
And you don't seem to under-stand, a shame you seemed an honest man

**Review by hyperioo — CHANGES_REQUESTED** [review]
Duplicate APC created at Guild Artist Office (91,85,2). Original at Guild Artist Office (94,84,2).
Duplicate APC created at Guild Artist Office (114,95,2). Original at Guild Artist Office (91,85,2).

+

For the holostool I'm not seeing anything that checks that it is still in holodeck area. Can I take it and run away with it to the other end of the ship?

**Comment by Humonitarian** [comment]
> Duplicate APC created at Guild Artist Office (91,85,2). Original at Guild Artist Office (94,84,2). Duplicate APC created at Guild Artist Office (114,95,2). Original at Guild Artist Office (91,85,2).
> 
> And
> 
> For the holostool I'm not seeing anything that checks that it is still in holodeck area. Can I take it and run away with it to the other end of the ship?

Like all holoitems, the console keeps a list of all holon items and deletes them when they leave the area. It does miss optimization and used process instead of signals, so I might look into it.

**Comment by hyperioo** [comment]
> > Duplicate APC created at Guild Artist Office (91,85,2). Original at Guild Artist Office (94,84,2). Duplicate APC created at Guild Artist Office (114,95,2). Original at Guild Artist Office (91,85,2).
> > And
> > For the holostool I'm not seeing anything that checks that it is still in holodeck area. Can I take it and run away with it to the other end of the ship?
> 
> Like all holoitems, the console keeps a list of all holon items and deletes them when they leave the area. It does miss optimization and used process instead of signals, so I might look into it.

Alright.
You still have the duplicate APC issue though.

**Comment by Humonitarian** [comment]
Drafting due to bugs

**Comment by Pink-Chink** [comment]
Marking as ready so I can put it on test

File: code/modules/holodeck/HolodeckControl.dm [hunk 1]
```diff
@@ -140,7 +141,7 @@
 
 /obj/machinery/computer/HolodeckControl/proc/update_projections()
 	if (safety_disabled)
-		item_power_usage = 2500
+		item_power_usage = 250
```
Comment (Humonitarian) [inline code/modules/holodeck/HolodeckControl.dm:144]:
```suggestion
		item_power_usage = 25
```

File: code/modules/holodeck/HolodeckControl.dm [hunk 2]
```diff
@@ -5,13 +5,13 @@
 	icon_screen = "holocontrol"
 
 	use_power = IDLE_POWER_USE
-	active_power_usage = 8000 //8kW for the scenery + 500W per holoitem
+	active_power_usage = 10000 //10kW for the scenery + 50W per holoitem
 
 	circuit = /obj/item/electronics/circuitboard/holodeckcontrol
 
-	var/item_power_usage = 500
+	var/item_power_usage = 50
```
Comment (Humonitarian) [inline code/modules/holodeck/HolodeckControl.dm:12]:
```suggestion
	var/item_power_usage = 5
```

File: code/modules/holodeck/HolodeckControl.dm [hunk 3]
```diff
@@ -5,13 +5,13 @@
 	icon_screen = "holocontrol"
 
 	use_power = IDLE_POWER_USE
-	active_power_usage = 8000 //8kW for the scenery + 500W per holoitem
+	active_power_usage = 10000 //10kW for the scenery + 50W per holoitem
```
Comment (Humonitarian) [inline code/modules/holodeck/HolodeckControl.dm:8]:
```suggestion
	active_power_usage = 5000 //5kW for the scenery + 5W per holoitem
```

**Review by hyperioo — APPROVED** [review]

**Pull request merged**